pyctbgui/services/ADC.py

```diff
@@ -26,6 +26,7 @@ def setup_ui(self):
 
     def initializeAllAnalogPlots(self):
         self.mainWindow.plotAnalogWaveform = pg.plot()
+        self.mainWindow.plotAnalogWaveform.addLegend(colCount=4)
```

Comment:
colCount=4 appears 4 times is this a coincidence or should it be a named variable?